modules/installation-configuration-parameters.adoc

```diff
@@ -474,6 +474,18 @@ Optional installation configuration parameters are described in the following ta
 |A PEM-encoded X.509 certificate bundle that is added to the nodes' trusted certificate store. This trust bundle may also be used when a proxy has been configured.
 |String
 
+|`capabilities`
+|Controls the installation of optional core cluster components.
+|String array
+
+|`capabilities.baselineCapabilitySet`
+|Selects an initial set of optional capabilities to enable. Valid values are `None`, `v4.11` and `vCurrent`. The default value is `vCurrent` which installs the recommended set of capabilities for the current version of {product-title}. In {product-title} {product-version}, the `baremetal` Operator, the `marketplace` Operator, and the `samples` content are optional capabilities.
+|String
+
+|`capabilities.additionalEnabledCapabilities`
+|Extends the set of optional capabilities beyond what you specify in `baselineCapabilitySet`. Valid values are `baremetal`, `marketplace` and `samples`. You may specify multiple capabilities in this parameter.
```

Comment:
supported values for `capabilities.additionalEnabledCapabilities` are `baremetal`, `marketplace`, `openshift-samples`, so need to update: `samples` -> `openshift-samples`